packages/block-editor/src/components/url-input/index.js

```diff
@@ -223,17 +223,27 @@ class URLInput extends Component {
 		this.inputRef.current.focus();
 	}
 
+	static getDerivedStateFromProps( { showSuggestionsOverride }, { showSuggestions } ) {
+		return {
+			showSuggestions: showSuggestionsOverride !== undefined ? showSuggestionsOverride : showSuggestions,
```

Comment:
Let's document this new prop in the component's README.

Reply:
I'll address this in a follow-up PR 👍